### DISCUSSION TITLE
addResource function for i18n-js

### DISCUSSION THREAD

**Discussion by Gabsys** [fnando/i18n-js#646] [Q&A] [answered]
Is there any built-in function available for i18n-js that works like `addResource()` of i18next so that we can programatically add translation strings?

**Comment by PikachuEXE** [comment, marked as answer]
For v3 I don't think so
Not sure about v4

You can modify the `translations` object directly though

**Reply by Gabsys** [reply]
Ok thanks ~~

**Reply by fnando** [reply]
v4 has `I18n#store`, which you can use to update the translations object.
